ogr/services/github.py

```diff
@@ -228,6 +228,55 @@ def get_fork(self, create: bool = True) -> Optional["GithubProject"]:
                 return None
         return self._construct_fork_project()
 
+    def get_owners(self) -> List[str]:
+        # in case of github, repository has only one owner
+        return [self.github_repo.owner.login]
+
+    def can_close_issue(self, username: str, issue: Issue) -> bool:
+        try:
+            collaborators = self._get_collaborators_with_permission()
+        except github.GithubException:
+            logger.debug(
+                f"{username} must have push access to view repository permissions."
+            )
+            return False
+
+        for login, permission in collaborators.items():
```

Comment:
Personally, I would like to see also the list/set of allowed users. (Since you are going through all of the collaborators and we are not saving any time with that.)

We have following possibilities:

1. `who_can_close_issues` and `can_close_issue` (which will use `who_can_close_issues` and checks the author of the issue) (We can move the `can_close_issue` to the Issue object later.)
2. `who_can_close_issue` with the optional parameter for specifying the issue id
3. leave it in the current state

I vote for the second one but make your own decision...;-)

(Same for the pull-requests.)